actioncable/app/assets/javascripts/action_cable/connection.coffee

```diff
@@ -90,18 +100,18 @@ class ActionCable.Connection
     open: ->
       ActionCable.log("WebSocket onopen event")
       @disconnected = false
```

Comment:
I tried this before, but that welcome message was coming in and making the connection monitor think it was still online. It doesn't happen when code is as it is.